server/config/passport-setup.js

```diff
@@ -7,7 +7,7 @@ passport.serializeUser((user, done) => {
 })
 
 passport.deserializeUser(async (id, done) => {
-    const user = await User.findByPk(id)
+    const user = await User.findByPk(id) || {}
     done(null, user.dataValues)
```

Comment:
why are we using empty object if user can't be found?

Reply:
safe check for line 11 coz user.dataValues but user from findByPk can be null